segmentation_labeling_app/transfers/utils.py

```diff
@@ -0,0 +1,188 @@
+import boto3
+from botocore.exceptions import ClientError
+import pathlib
+import sqlite3
+import json
+import jsonlines
+import tempfile
+
+
+def upload_file(file_name, bucket, key=None):
+    """Upload a file to an S3 bucket
+
+    Parameters
+    ----------
+    file_name: path-like object
+        full path to local file
+    bucket: str
+        name of bucket
+    key: str
+        object key. If not passed, object key will be
+        basename of the file_name
+
+    Returns
+    -------
+    s3_uri : str
+        URI to S3 object just uploaded
+
+
+    """
+    if key is None:
+        key = pathlib.PurePath(file_name).name
+
+    client = boto3.client('s3')
+    client.upload_file(file_name, bucket, key)
```

Comment:
As per slack discussion, it is probably best practice to include a round-trip download and contents check here.

Reply:
included roundtrip check